src/broker/content-cache.c

```diff
@@ -963,4 +995,4 @@ struct content_cache *content_cache_create (flux_t *h, attr_t *attrs)
 
 /*
  * vi:tabstop=4 shiftwidth=4 expandtab
- */
+ */
```

Comment:
Please fix trailing white space (I'll just call out this one - there are some others).